vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +96,126 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
+     * exhibitionTitles list for each user selected Exhibition Language.
+    */
+    public void setAboutPageTitle(LocalizedTextForm title, ExhibitionAboutPage exhibitionAboutPage) {
+        if(title!=null) {
+            
+            LocalizedText localizedText = localizedTextRepo.findById(title.getLocalisedTextId()).orElse(null);
+            if(localizedText != null) {            
+                localizedText.setText(title.getText());            
+            } else {
+                
+                ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findById(title.getExhibitionLanguageId()).orElse(null);
+                if(exhibitionLanguage != null) {
+                    localizedText = new LocalizedText(exhibitionLanguage, title.getText());
+                    exhibitionAboutPage.getExhibitionTitles().add(localizedText);
+                    exhibitionLanguage.getLocalizedTexts().add(localizedText);
+                }
+            }
+            
+        }
+
+    }
+    
+    /**
+     * This method maps the description in ExhibitionAboutPage, and add that to
+     * exhibitionTextDescriptions list for each user selected Exhibition Language.
+    */
+    public void setAboutPageDescription(LocalizedTextForm aboutPageText, ExhibitionAboutPage exhibitionAboutPage) {
+        if(aboutPageText!=null) {
+            LocalizedText localizedText = localizedTextRepo.findById(aboutPageText.getLocalisedTextId()).orElse(null);
+            
+            if(localizedText != null) {
+                localizedText.setText(aboutPageText.getText());
+            } else {
+                ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findById(aboutPageText.getExhibitionLanguageId()).orElse(null);
+                if(exhibitionLanguage != null) {
+                    exhibitionAboutPage.getExhibitionTextDescriptions().add(new LocalizedText(exhibitionLanguage, aboutPageText.getText()));
+                }
+            }        
+        }
+    }
+       
+    /**
+     * Creates About Page form object
+     */
+    @Override
+    public AboutPageForm createAboutPageForm() {
+        ExhibitionAboutPage exhibitionAboutPage = getExhibitionAboutPage(); 
+
+        AboutPageForm aboutPageForm=new AboutPageForm();
+        aboutPageForm.setAboutPageText(exhibitionAboutPage.getAboutPageText());
+        aboutPageForm.setTitle(exhibitionAboutPage.getTitle());        
+        
+        IExhibition startExhibtion = exhibitionManager.getStartExhibition();
+        
+        
+        
+        IExhibitionLanguage defaultLanguage = exhibitionManager.getDefaultLanguage(startExhibtion);
+                
+               
+        
+        aboutPageForm.getTitles().add(createLocalizedTitleForm(exhibitionAboutPage, defaultLanguage));
+        
+        aboutPageForm.getAboutPageTexts().add(createLocalizedAboutTextForm(exhibitionAboutPage, defaultLanguage));
```

Comment:
why are these extra and not added in the loop?

Reply:
The default language is added first to ensure it shows up first in the ui. Is there any other way to do this?

Reply:
This seems wonky. Maybe just have a default language in the about form and a list of other languages?